site/content/en/docs/Advanced/multi-cluster-allocation.md

```diff
@@ -38,9 +38,11 @@ spec:
 EOF
 ```
 
-To define the local cluster priority, similarly, an allocation rule should be defined, while leaving allocationEndpoints unset. If the local cluster priority is not defined, the allocation from the local cluster happens only if allocation from other clusters with the existing allocation rules is unsuccessful.
+To define the local cluster priority an allocation rule should be defined _without_ an `allocationEndpoints` field. If the local cluster priority is not defined, the allocation from the local cluster happens only if allocation from other clusters with the existing allocation rules is unsuccessful. A local cluster priority must be defined if using multi-cluster in only a single cluster (e.g. in development).
```

Comment:
That's incorrect, it doesn't work like that in actual practice. If it _should_ be working like that, that's an additional bug that needs to be resolved.

```
$ kubectl get gsap
No resources found in default namespace.

$ curl --key ${KEY_FILE} --cert ${CERT_FILE} --cacert ${TLS_CA_FILE} -H "Content-Type: application/json" --data '{"namespace":"'${NAMESPACE}'", "multiClusterSetting":{"enabled":true}}' https://${EXTERNAL_IP}/gameserverallocation -XPOST
{"error":"no multi-cluster allocation policy is specified","code":2,"message":"no multi-cluster allocation policy is specified"}
```

Reply:
Thanks for providing the context. If you enable the feature `multiClusterSetting.enable=true` on your allocation request, you need to provide an allocation policy in the cluster. Otherwise, leave `multiClusterSetting` unset and you will not get the error.

Reply:
That's why I added this line in the first place: I want to test with `multiClusterSetting.enable=true` in local dev (since that's what my production code is going to have), therefore I needed to add a GSAP for the single cluster, therefore I added this line to the docs. I'm going to leave this line in and consider this comment resolved in light of that.

Reply:
Thanks. This clarifies it. Can you please add a clarification sentence at the end such as "otherwise, the request fails with the error "no multi-cluster allocation policy is specified"?

Reply:
Done.